packages/editor/src/components/post-locked-modal/index.js

```diff
@@ -155,38 +159,37 @@ class PostLockedModal extends Component {
 						<div>
 							{ userDisplayName ?
 								sprintf(
-									/* translators: 'post' is generic and may be of any type (post, page, etc.). */
-									__( '%s now has editing control of this post. Don\'t worry, your changes up to this moment have been saved' ),
+									/* translators: %s: user's display name */
+									__( '%s now has editing control of this post. Don’t worry, your changes up to this moment have been saved.' ),
```

Comment:
I think we want to keep the slashed single quote, the there a reason that was changed here?
```suggestion
									__( '%s now has editing control of this post. Don\'t worry, your changes up to this moment have been saved.' ),
```

Reply:
An apostrophe is the correct character to use in English here, not a single quote. It's certainly typographically better; that said, we should just be consistent and use one or the other everywhere. See: #7555.